app/views/home/home.html.erb

```diff
@@ -101,7 +101,7 @@
     <img class="img img-circle" src="//i.publiclab.org/system/images/photos/000/006/671/original/canoe.jpg"/>
     <h2 class="the-problem"> <%= t('home.home.the_problem.title') %> </h2>
     <h4><%= t('home.home.the_problem.lack_access') %> <a href="/tools"><%= t('home.home.the_problem.tools') %></a> <%= t('home.home.the_problem.need_participation') %></h4>
-    <p><a href="/blog" class="btn btn-primary btn-lg">Read stories &raquo;</a></p>
+    <p><a href="/wiki/stories" class="btn btn-primary btn-lg">Read blog stories &raquo;</a></p>
```

Comment:
This route doesn't seem to exist in `config/routes.rb`

Reply:
@jywarren

Reply:
True - we may eventually move this into the Features system; but for now this looks great!